File: python/BarangElektronik.py
class BarangElektronik:
    def __init__(self, id, nama, jenis, merek, harga):
        self.id = id
        self.nama = nama
        self.jenis = jenis
        self.merek = merek
        self.harga = harga

    # Fungsi Getter 
    def get_id(self):
        return self.id

    def get_nama(self):
        return self.nama

    def get_jenis(self):
        return self.jenis

    def get_merek(self):
        return self.merek

    def get_harga(self):
        return self.harga
    
    # Fungsi Setter
    def set_nama(self, nama):
        self.nama = nama

    def set_jenis(self, jenis):
        self.jenis = jenis

    def set_merek(self, merek):
        self.merek = merek

    def set_harga(self, harga):
        self.harga = harga

File: php/BarangElektronik.php
<?php

// =================================================================
// Class BarangElektronik
// =================================================================
class BarangElektronik {
    private $id;
    private $nama;
    private $jenis;
    private $merek;
    private $harga;
    private $gambar;

    public function __construct($id, $nama, $jenis, $merek, $harga, $gambar) {
        $this->id = $id;
        $this->nama = $nama;
        $this->jenis = $jenis;
        $this->merek = $merek;
        $this->harga = $harga;
        $this->gambar = $gambar;
    }

    // Fungsi Get
    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getJenis() { return $this->jenis; }
    public function getMerek() { return $this->merek; }
    public function getHarga() { return $this->harga; }
    public function getGambar() { return $this->gambar; }
    
    // Fungsi Set
    public function setNama($nama) { $this->nama = $nama; }
    public function setJenis($jenis) { $this->jenis = $jenis; }
    public function setMerek($merek) { $this->merek = $merek; }
    public function setHarga($harga) { $this->harga = $harga; }
    public function setGambar($gambar) { $this->gambar = $gambar; }
}

?>